src/main/kotlin/io/github/orangain/prettyjsonlog/logentry/Timestamp.kt

```diff
@@ -39,7 +39,7 @@ sealed interface Timestamp {
     }
 }
 
-private val timestampKeys = listOf("timestamp", "time", "@timestamp")
+private val timestampKeys = listOf("timestamp", "time", "@timestamp", "ts", "T")
```

Comment:
According to the zap document and your explanation, the key `T` is only used in the DevelopmentConfig, and its default encoding is `console` not `json`. I guess the key `T` is rarely used in the JSON format. If my understanding is true, please remove `T` from the `timestampKeys`.

I would only want to include keys that are used in the real world JSON logs.

Reply:
> I would only want to include keys that are used in the real world JSON logs.

Are there any plans to add configuration for the plugin to allow end users to add their own values?

I feel like it would be pretty annoying to come across this and see support for zap was added, but development mode was excluded. 
And I don't believe there is any real downside to having extra keys, regardless of the rarity of its usage?

Reply:
> Are there any plans to add configuration for the plugin to allow end users to add their own values?

There is no immediate plan, but I will consider it for the future if there is a demand for it.

> I feel like it would be pretty annoying to come across this and see support for zap was added, but development mode was excluded.

To be honest, as an application developer, I believe that there is no strong motivation to use different JSON keys for the development and the production.

If a developer uses zap with default configuration `zap.NewDevelopment()`, logs are written in the human-readable text format, not in the JSON format. The plugin is not necessary for this case.

```
2024-08-14T12:35:10.203+0900	INFO	try-uber-go-zap-log/main.go:10	application starting...
```

In order to write logs in the JSON format while development, there are two options.

**Option 1: ProductionConfig with development mode**

```go
	config := zap.NewProductionConfig()
	config.Development = true
	if logger, err := config.Build(); err == nil {
		logger.Info("application starting...")
	}
```

In this case, logs are written in the JSON format. This case is now supported by the plugin thanks to your contribution. 

```
{"level":"info","ts":1723607766.200156,"caller":"try-uber-go-zap-log/main.go:19","msg":"application starting..."}
```

**Option 2: DevelopmentConfig with json encoding**

```go
	config := zap.NewDevelopmentConfig()
	config.Encoding = "json"
	if logger, err := config.Build(); err == nil {
		logger.Info("application starting...")
	}
```

In this case, logs are still written in the JSON format, but this JSON log seems less readable than the option 1. I believe that developers prefer the option 1 to the option 2.

```
{"L":"INFO","T":"2024-08-14T12:37:16.025+0900","C":"try-uber-go-zap-log/main.go:12","M":"application starting..."}
```

In addition, I guess the following comment in [the source code](https://github.com/uber-go/zap/blob/3f27eb9c787ea9b2ddfc1f707ef291c9070655fe/config.go#L202) implies that these keys are not intended to be used as a JSON key.

> // Keys can be anything except the empty string.


----

> And I don't believe there is any real downside to having extra keys, regardless of the rarity of its usage?

You are right. However, if we continue to add more and more keys in the future that are rarely used in that way, it will affect performance and make maintenance more difficult.